test/views/coronavirus_local_restrictions/results_html_test.rb

```diff
@@ -0,0 +1,169 @@
+require "test_helper"
+require "gds_api/test_helpers/mapit"
+require_relative "../../support/coronavirus_local_restrictions_helpers"
+
+class ResultsHtmlTest < ActionView::TestCase
+  include GdsApi::TestHelpers::Mapit
+  include CoronavirusLocalRestrictionsHelpers
+  helper Rails.application.helpers
+
+  describe "current restrictions" do
+    test "rendering tier 4 results for a postcode in tier 4" do
+      render_tier_results(current_alert_level: 4)
+
+      assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_four.heading_pretext")
+      assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_four.heading_tier_label")
+      assert_includes rendered, area
+    end
+
+    test "rendering tier 3 results for a postcode in tier 3" do
+      render_tier_results(current_alert_level: 3)
+
+      assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_three.heading_pretext")
+      assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_three.heading_tier_label")
+      assert_includes rendered, area
+    end
+
+    test "rendering tier 2 results for a postcode in tier 2" do
+      render_tier_results(current_alert_level: 2)
+
+      assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_two.heading_pretext")
+      assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_two.heading_tier_label")
+      assert_includes rendered, area
+    end
+
+    test "rendering tier 1 results for a postcode in tier 1" do
+      render_tier_results(current_alert_level: 1)
+
+      assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_one.heading_pretext")
+      assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_one.heading_tier_label")
+      assert_includes rendered, area
+    end
+
+    test "rendering postcode match if the postcode is correct except for a special character" do
+      area = "Coruscant Planetary Council"
+      postcode = ".E1 8QS"
+
+      stub_local_restriction(postcode: "E1 8QS", name: area, current_alert_level: 2)
+
+      @search = PostcodeLocalRestrictionSearch.new(postcode)
+
+      view.stubs(:out_of_date?).returns(false)
+
+      render template: "coronavirus_local_restrictions/results"
+
+      assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_two.heading_pretext")
+      assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_two.heading_tier_label")
+      assert_includes rendered, area
+    end
+  end
+
+  describe "devolved nations" do
+    test "rendering results for a Welsh postcode" do
+      postcode = "LL11 0BY"
```

Comment:
does this postcode impact the test at all? I'm wondering if it's superfluous.

Reply:
ah yeah it doesn't! Thanks, have removed.